plaidml/bridge/tensorflow/tests/BUILD

```diff
@@ -0,0 +1,109 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+load("@org_tensorflow//tensorflow:tensorflow.bzl", "tf_cc_test")
+
+cc_library(
+    name = "codegen_test",
+    testonly = True,
+    srcs = ["codegen_test.cc"],
+    hdrs = ["codegen_test.h"],
+    deps = [
+        "//plaidml:testenv",
+        "//plaidml/bridge/tensorflow/service:compiler",
+        "@org_tensorflow//tensorflow/compiler/xla/tests:hlo_test_base",
+        "@org_tensorflow//tensorflow/core:test",
+        "@org_tensorflow//tensorflow/core:test_main",
+    ],
+)
+
+cc_library(
+    name = "filecheck",
+    testonly = True,
+    srcs = ["filecheck.cc"],
+    hdrs = ["filecheck.h"],
+    data = [
+        "@llvm-project//llvm:FileCheck",
+    ],
+    deps = [
+        "@org_tensorflow//tensorflow/compiler/xla:statusor",
+        "@org_tensorflow//tensorflow/compiler/xla:types",
+        "@org_tensorflow//tensorflow/compiler/xla:util",
+        "@org_tensorflow//tensorflow/core:lib",
+	"@bazel_tools//tools/cpp/runfiles",
+    ],
+)
+
+tf_cc_test(
+    name = "compare_op_test",
+    srcs = ["compare_op_test.cc"],
+    extra_copts = ["-fexceptions"],
+    deps = [
+        ":codegen_test",
+        ":filecheck",
+        "//plaidml/bridge/tensorflow/service:compiler",
+        "@com_google_absl//absl/strings",
+        "@com_google_googletest//:gtest",
+        "@org_tensorflow//tensorflow/compiler/xla/service:hlo",
+        "@org_tensorflow//tensorflow/compiler/xla/tests:hlo_test_base",
+        "@org_tensorflow//tensorflow/compiler/xla/tests:test_utils",
+        "@org_tensorflow//tensorflow/core:framework",
+        "@org_tensorflow//tensorflow/core:lib",
+        "@org_tensorflow//tensorflow/core:test",
+        "@org_tensorflow//tensorflow/core:test_main",
+    ],
+)
+
+py_binary(
+    name = "conv_op_test_gen",
+    srcs = ["conv_op_test_gen.py"],
+    testonly = 1,
+)
+
+genrule(
+    name = "conv_op_test_genrule",
+    srcs = [],
+    outs = ["conv_op_test.h.inc"],
+    cmd = "XLA_FLAGS=\"--xla_dump_to=/tmp/hlo_module\" TF_XLA_FLAGS=\"--tf_xla_auto_jit=2 --tf_xla_cpu_global_jit --tf_xla_min_cluster_size=0\" $(location :conv_op_test_gen) --output $@",
```

Comment:
See other comment; this test runs fine even when these flags are removed here.

Reply:
I think I prefer specifying them in the `BUILD` file rather than the `bazelrc`, but I'm open to hearing any thoughts on why I should do it the other way around.

Reply:
No I agree; I think having them in the BUILD file makes more sense since those flags are only really needed for the testing (as far as I'm aware)